cc.cubocore.CoreTime.yml

```diff
@@ -0,0 +1,44 @@
+app-id: cc.cubocore.CoreTime
+runtime: org.kde.Platform
+runtime-version: '6.8'
+sdk: org.kde.Sdk
+command: coretime
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --share=network # Required to show weather info
+  - --socket=wayland
+  - --filesystem=home
+  - --filesystem=host
+  - --socket=pulseaudio # Required for ringing alarm
+  - --device=dri
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --talk-name=org.freedesktop.Notifications
+  - --own-name=org.kde.* # For showing in the system tray
```

Comment:
the system tray doesn't work with this. Has never.

Reply:
it's also blocked in linter, so you can't use this.